doc/WebOnChain.plantuml

```diff
@@ -0,0 +1,186 @@
+@startuml
+participant "End user" as user
+participant "Browser" as ui
+participant "Thyra server" as thyra
+participant "Hosting websites" as web
+participant "Wallet" as w
+participant "DNS" as dns
+participant "Event Manager" as em
+participant "Storage" as storage
+participant "Node (API)" as node
+
+user -> ui: click on install 'Web on-chain' plug-in
+ui -> thyra: GET /plugin/webOnChain
+thyra -> web: GET /plugin/webOnChain
+web -> thyra: OK response
+thyra -> ui: display landing page WebOnChain.massa
+ui -> user: display landing page WebOnChain.massa
+
+user -> ui: Click on "websites" page
+ui -> thyra: GET /plugin/wallet
+thyra -> web: GET /plugin/wallet
+web -> thyra: GET /plugin/wallet/{status}
+thyra -> w: GET /plugin/wallet/{status}
+w -> w: is wallet status active?
```

Comment:
What appens if there is more than one wallets ? no wallet ?

Reply:
ack, will suggest something to handle that.